package.json

```diff
@@ -27,14 +27,14 @@
     "nyc": "^11.0.0",
     "pump": "^3.0.0",
     "rimraf": "^2.6.2",
-    "tap": "^11.0.0",
+    "tap": "^12.0.0",
```

Comment:
Is there a reason we can't update to latest (v14)?

Reply:
it does not support Node 6